job_runner/reporting/file.py

```diff
@@ -32,42 +37,56 @@ def __init__(self, sync_disc_timeout=10):
         self.node = socket.gethostname()
         self._sync_disc_timeout = sync_disc_timeout
 
-    def report(self, msg):
+    def report(self, msg: Message):
         job_status = {}
+
+        def job_status_insert(**kwargs) -> None:
+            for key, value in kwargs.items():
+                job_status[key] = value
+
         if msg.job:
-            index = msg.job.index
-            job_status = self.status_dict["jobs"][index]
+            logger.debug("Adding message job to status dictionary.")
```

Comment:
It might be warranted, but do you think we need this many debug statements?

Reply:
I was told at a standup that it is better to have too many and later with the team reduce to what is really needed